Add binary to benchmark model load speed

SS-JIA opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#74700 Add binary to benchmark model load speed**
* #74769 [vulkan] Add option to trigger Vulkan context to load on application start
* #74699 [vulkan] Remove unnecessary include in vulkan_api_test

Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

SS-JIA committed:
Add binary to benchmark model load speed

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

beback4u commented on binaries/load_benchmark_torch.cc:
```diff
@@ -0,0 +1,95 @@
+#include <vector>
+
+#include <ATen/ATen.h>
+#include "caffe2/core/timer.h"
```
Can we re-organize the order of includes?

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

Commit:
Update on "Add binary to benchmark model load speed"


Differential Revision: [D35124881](https://our.internmc.facebook.com/intern/diff/D35124881)

[ghstack-poisoned]

SS-JIA commented:
@SS-JIA has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D35124881).
